### ISSUE TITLE
toggleEmailInput function call causes unnecessary PUT request

### ISSUE THREAD

**Issue by CristinaSolana** [envistaInteractive/itagroup-ecommerce-template#383]
On both the index and profile-edit, toggleEmailInput(checkBox, email); is called and does an unnecessary PUT request on page load. This function should only be called when there is a change to the email newsletter fields.

**Comment by itsDiwaker** [comment]
@CristinaSolana while fixing the issue reported found another issue. 
I think backend needs a logic update to fix this. 
Here's the screencast: https://drive.google.com/file/d/1e7Vqf6-JYVJx6_tifvr60appfoGAbCJL/view

**Comment by CristinaSolana** [comment]
@itsDiwaker Yes, this is an issue on the backend side. If it cannot be resolved on the backend because of architectural limitation, we will need to fix on our side by modifying existing attributes object and sending entire obj in request.
